examples/osp-ubuntu.yaml

```diff
@@ -157,6 +159,28 @@ spec:
       rm -f "$cni_filename"
       cd -
 
+      {{- /* # cri-tools variables */}}
```

Comment:
I don't this is required. This is only used to download crictl tooling which should users decide to have.

Reply:
In MC we install CRI-TOOLS in all instances, therefore I ported it here. If a user wants to get rid of it, can simply create his own OSP without this part.

Reply:
While I am not a big fan of adding things in the primitive configurations that needed for debugging for example. However I have nothing against this tool :D